quote-book/synchronized/src/lib.rs

```diff
@@ -126,52 +404,93 @@ mod tests {
         ledger
     }
 
-    #[test]
-    fn basic_happy_flow() {
+    fn get_remove_quote_callback() -> RemoveQuoteCallback {
+        Arc::new(Mutex::new(|_| {
+            //Do nothing
```

Comment:
i think we should try to do something better here, for instance, we could have a `Vec<QuoteId>` and then in the test assert that the right things were reported as removed.